design/tlog-spilling.md.html

```diff
@@ -225,7 +225,7 @@
 That document describes a `log_version` configuration setting that controls the
 availability of new transaction log features.  A similar configuration setting
 was created, `log_spill`, that at `log_version>=3`, one may `fdbcli>
-configure log_spill:=2` to enable spill-by-reference.  Only FDB 6.1 or newer
+configure log_spill:=2` to enable spill-by-reference.  Only FDB 6.1 or older
```

Comment:
I think the original meaning is 6.1 or later versions, so the change is incorrect. Please revert.

Reply:
Ok, but just to be sure, did you notice the next sentence? "FDB 6.2 will use spill-by-reference by default" If this says FDB 6.1 or newer, I am a bit confused.

Reply:
Yes, that is correct. We usually introduce a feature in one version, e.g., 6.1 in this case, and in the next version (6.2) can make it the default. This is intentional so that there is compatibility between two consecutive versions, i.e., 6.0 to/from 6.1, and 6.1 to/from 6.2.

Reply:
Oh I see. Reverting now. Thank you ^-^